ui/src/app/views/workspace/component.css

```diff
@@ -0,0 +1,52 @@
+
```

Comment:
I chatted with David about using SCSS [I think it was] for CSS pre-processing. We'll probably want to switch to that fairly soon, but that may be a good one to ignore until post-F2F. So, just FYI.